packages/flutter/lib/src/material/divider.dart

```diff
@@ -117,9 +117,21 @@ class Divider extends StatelessWidget {
   /// ```
   /// {@end-tool}
   static BorderSide createBorderSide(BuildContext context, { Color color, double width }) {
```

Comment:
We should document what happens if BuildContext is null. This is a bit of an odd method; usually this kind of thing would require a non-null context or else context would be a named parameter.

Reply:
done